website/docs/vs-code-snippets/python.json

```diff
@@ -293,5 +293,81 @@
             "$0"
         ],
         "description": "Set up ScriptRunConfig including compute target, environment and experiment"
```

Comment:
Should we start adding Docs to the snippet Descriptions? For this one I didn't really understand command until I read the docs. https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.script_run_config.scriptrunconfig?view=azure-ml-py#constructor

Reply:
Actually no worries. Adding docs is not necessary. If we need to add docs link, we need to add to all snippets